### ISSUE TITLE
Null Safety

### ISSUE THREAD

**Issue by BartusZak** [arnemolland/sigv4#25]
Hey @arnemolland,
Do you plan to update package to support Dart Null Safety?

```
dart pub outdated --mode=null-safety
```
```
Resolving...
Showing dependencies that are currently not opted in to null-safety.
[✗] indicates versions without null safety support.
[✓] indicates versions opting in to null safety.   

Computing null safety support...
Package Name                          Current        Upgradable      Resolvable  Latest

direct dependencies:
sigv4                                 ✗4.1.1         ✗4.1.2          -           ✗4.1.2
```

**Comment by arnemolland** [comment]
Hi @BartusZak, migrating should be pretty simple, but at the moment this library is blocked by chopper and partly dio (only pre-release null-safety). I will try to release a pre-release temporarily dropping the chopper extension shortly!

**Comment by BartusZak** [comment]
Love to hear that! 
Thanks for all your work.

**Comment by arnemolland** [comment]
I've published it as a 5.0.0-nullsafety.0 pre-release until the Dio migration is in stable.
